.github/workflows/deploy-dev.yml

```diff
@@ -0,0 +1,23 @@
+name: Deploy to Dev
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: environment
+        required: true
+        default: dev
```

Comment:
Is this allowing you to specify an environment when manually invoking this? Is this input actually used? (Looks like `jobs.deploy` just uses `dev`).

Reply:
Hmm. I actually don't know! Let's try without it and see